src/Data/Fin/Properties.agda

```diff
@@ -65,6 +65,9 @@ private
 ¬Fin0 : ¬ Fin 0
 ¬Fin0 ()
 
+nonZeroIndex : Fin n → ℕ.NonZero n
```

Comment:
Missing in CHANGELOG?

Reply:
Now: after resolving merge conflict, it's [L37 in `CHANGELOG`](https://github.com/agda/agda-stdlib/blob/fac8e6c5f7cafddba653b5a3d150ed170e758f9a/CHANGELOG.md?plain=1#L37)